integration_tests/cdk/requirements.txt

```diff
@@ -0,0 +1,15 @@
+aws-cdk-lib>=2.99.1
+aws_cdk.aws_cognito_identitypool_alpha>=2.99.0a0
+aws-cdk.aws-apigatewayv2-alpha>=2.99.0a0
+constructs>=10.0.0,<11.0.0
+pydantic==2.0.2
+pydantic-settings==2.0.1
+black==22.3.0
+boto3==1.24.15
+boto3-stubs[cognito-idp,cognito-identity]
+flake8==4.0.1
```

Comment:
flake and black are dev dependencies

Reply:
just removed them ; we have this 'tox' workflow for linting/formatting.